src/main/scala/Parser.scala

```diff
@@ -3,15 +3,17 @@ import scopt.Read
 import java.text.SimpleDateFormat
 import java.util.Date
 import scala.util.matching.Regex
+import scala.compiletime.ops.string
 
 // Recursive funtion for processing tasks
 object Parser {
 
   def apply(args: Seq[String]) =
     OParser.parse(parser, args, Config()) match
       // This is where our task object will end up as part of the config object
-      case Some(config) => config
-      case _            => None
+      case Some(config) => println(config)
+      case _            => println("PLEASE ENTER ARGS: pg task --add name=hello,description=world,effort=12,due=2020-02-12"
```

Comment:
The config object holds the config for all args, not just the args for tasks. If any of the other commands fail, this message is still used.